cms/templates/group_configurations.html

```diff
@@ -33,31 +31,35 @@ <h1 class="page-header">
         <small class="subtitle">${_("Settings")}</small>
         <span class="sr">&gt; </span>${_("Group Configurations")}
       </h1>
-      <nav class="nav-actions">
-        <h3 class="sr">${_("Page Actions")}</h3>
-        <ul>
-          <li class="nav-item">
-            <a href="#" class="button new-button"><i class="icon-plus"></i> ${_("New Group Configuration")}</a>
-          </li>
-        </ul>
-      </nav>
     </header>
   </div>
 
   <div class="wrapper-content wrapper">
     <section class="content">
       <article class="content-primary" role="main">
-      % if configurations is None:
-        <div class="notice notice-incontext notice-moduledisabled">
-            <p class="copy">
-                ${_("This module is disabled at the moment.")}
-            </p>
+        <div class="wrapper-groups cohort-groups">
+          <h3 class="title">${_("Cohorted Content")}</h3>
```

Comment:
@dan-f Per conversation with @explorerleslie this heading should be changed to "Cohorted Content Groups" so that it's parallel with "Experiment Groups" below.